app/views/unattended/provisioning_templates/iPXE/autoyast_default_ipxe.erb

```diff
@@ -24,15 +24,11 @@ ping --count 1 ${netX/gateway} || echo Ping to Gateway failed or ping command no
 echo Trying to ping DNS: ${netX/dns}
 ping --count 1 ${netX/dns} || echo Ping to DNS failed or ping command not available.
 
-<% boot_files_uris = @host.operatingsystem.boot_files_uri(medium_provider) -%>
-<% kernel = boot_files_uris[0] -%>
-<% initrd = boot_files_uris[1] -%>
-
 <% subnet = @host.subnet -%>
 <% if subnet.respond_to?(:dhcp_boot_mode?) && subnet.dhcp_boot_mode? -%>
-kernel <%= kernel %> initrd=initrd.img splash=silent install=<%= medium_uri %> autoyast=<%= foreman_url('provision') %> text-mode=1 useDHCP=1 <%= extra_args.join(' ')%>
+kernel <%= @kernel_uri %> initrd=<%= @initrd_uri.split('/').last %> splash=silent install=<%= medium_uri %> autoyast=<%= foreman_url('provision') %> text-mode=1 useDHCP=1 <%= extra_args.join(' ')%>
```

Comment:
Overall, pretty cool cleanup!

I would prefer to have a method which returns @initrd_file and can be used in the various locations

Reply:
Makes sense. I wasn't allowed to use `File.basename` but given the repetition I don't think that woud have been pretty anyway

Reply:
would be nice to have this including the new initrd_file method.